packages/experimental/Button/src/FabButton/FabButton.tsx

```diff
@@ -0,0 +1,24 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { View } from 'react-native';
+import { fabButtonName, FabButtonType } from './FabButton.types';
+import { Text } from '@fluentui-react-native/experimental-text';
+import { compose, UseSlots } from '@fluentui-react-native/framework';
+import { Icon } from '@fluentui-react-native/icon';
+import { ButtonCoreProps } from '../Button.types';
+
+export const FabButton = compose<FabButtonType>({
+  displayName: fabButtonName,
+  slots: {
+    root: View,
+    icon: Icon,
+    content: Text,
+  },
+  render: (_userProps: ButtonCoreProps, _useSlots: UseSlots<FabButtonType>) => {
+    return (_final: ButtonCoreProps, ..._children: React.ReactNode[]) => {
+      return null;
+    };
+  },
+});
+
+export default FabButton;
```

Comment:
FAB == Floating Action Button, so FAB Button is Floating Action Button Button. 
Maybe just name it FAB? Both react-native-elements and react-native-paper  (the #1 and #2 RN UI frameworks by usage) name is FAB too.
https://reactnativeelements.com/docs/fab
https://callstack.github.io/react-native-paper/fab.html

Reply:
Sounds good, done with latest commits.